### PR TITLE
fix: Improved SEO

### PR DESCRIPTION
**Pull request by parasss19**
## Description
- Added proper meta tags to improve the SEO score
- Also properly added keywords , titles, description etc to improve SEO

### CODE REVIEW

**Comment by parasss19** [comment]
Hey @priyankarpal Please review it
- I closed my previous PR as it contain so many conflicts
- This is my new PR for same issue with all the changes you wanted

File: app/layout.tsx [hunk 1]
```diff
@@ -19,6 +19,30 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
+      <head>
+        <meta name="title" content="ProjectHut-Learn.Build.Share" />
```
Comment (priyankarpal) [inline app/layout.tsx:23]: change here 
```suggestion
        <meta name="title" content={metadata.description} />
```

File: app/layout.tsx [hunk 2]
```diff
@@ -19,6 +19,30 @@ export default function RootLayout({
...
+        <meta
+          name="description"
+          content="ProjectsHut is an open source web app that enables users to freely publish their projects and create user profiles within the platform "
```
Comment (priyankarpal) [inline app/layout.tsx:26]: same here
```suggestion
          content={metadata.description}
```

File: app/layout.tsx [hunk 3]
```diff
@@ -19,6 +19,30 @@ export default function RootLayout({
...
+        <meta property="og:title" content={metadata.title} />
+        <meta property="og:description" content={metadata.description}  />
+        <meta property="og:url" content="https://projectshut.vercel.app/" /> 
+        <meta property="og:type" content="website" />
+        <meta property="og:site_name" content="ProjectHut" />
+        {/* Twitter meta tags for website preview */}
+        <meta name="twitter:card" content="summary_large_image" />
+        <meta property="twitter:title" content={metadata.title} />
+        <meta property="twitter:description" content={metadata.description} />
+        <meta property="twitter:image" content="" /> {/* Replace with the actual preview image URL */}
```
Comment (priyankarpal) [inline app/layout.tsx:44]: add the image url 
```suggestion
        <meta property="twitter:image" content="https://user-images.githubusercontent.com/88102392/251507361-e71396f9-92c6-4664-b7e4-4275cb902e65.png" /> 
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by parasss19** [comment]
Hey @priyankarpal Please review it now

**Review by priyankarpal — APPROVED** [review]

**Pull request merged**